docs/4.2/features/enhanced_session_recording.md

```diff
@@ -243,4 +250,60 @@ ssh_service:
 ```
 
 ## 5. Inspect Logs
-The resulting enhanced session recording will be shown in the logs. 
\ No newline at end of file
+The resulting enhanced session recording will be shown in [Teleports Audit Log](https://gravitational.com/teleport/docs/architecture/teleport_auth/#audit-log).
+
+
+
+```bash
+$ teleport-auth ~:  tree /var/lib/teleport/log
+/var/lib/teleport/log
+├── 1048a649-8f3f-4431-9529-0c53339b65a5
+│   ├── 2020-01-13.00:00:00.log
+│   └── sessions
+│       └── default
+│           ├── fad07202-35bb-11ea-83aa-125400432324-0.chunks.gz
+│           ├── fad07202-35bb-11ea-83aa-125400432324-0.events.gz
+│           ├── fad07202-35bb-11ea-83aa-125400432324-0.session.command-events.gz
+│           ├── fad07202-35bb-11ea-83aa-125400432324-0.session.network-events.gz
+│           └── fad07202-35bb-11ea-83aa-125400432324.index
+├── events.log -> /var/lib/teleport/log/1048a649-8f3f-4431-9529-0c53339b65a5/2020-01-13.00:00:00.log
+├── playbacks
+│   └── sessions
+│       └── default
+└── upload
+    └── sessions
+        └── default
+```
+
+To quickly check the status of the audit log. You can simply tail the logs with 
+`tail -f /var/lib/teleport/log/events.log`, the resulting capture from Teleport will 
+be a JSON log for each command and network request. 
+
+```
+{"argv":["google.com"],"cgroup_id":4294968064,"code":"T4000I","ei":5,"event":"session.command","login":"root","namespace":"default","path":"/bin/ping","pid":2653,"ppid":2660,"program":"ping","return_code":0,"server_id":"96f2bed2-ebd1-494a-945c-2fd57de41644","sid":"44c6cea8-362f-11ea-83aa-125400432324","time":"2020-01-13T18:05:53.919Z","uid":"734930bb-00e6-4ee6-8798-37f1e9473fac","user":"benarent"}
+```
+
+Formatted the resulting data will look like. 
+```json 
+{ 
+"argv":[ 
+"google.com"
+],
+"cgroup_id":4294968064,
+"code":"T4000I",
+"ei":5,
+"event":"session.command",
+"login":"root",
+"namespace":"default",
+"path":"/bin/ping",
+"pid":2653,
+"ppid":2660,
+"program":"ping",
+"return_code":0,
+"server_id":"96f2bed2-ebd1-494a-945c-2fd57de41644",
+"sid":"44c6cea8-362f-11ea-83aa-125400432324",
+"time":"2020-01-13T18:05:53.919Z",
+"uid":"734930bb-00e6-4ee6-8798-37f1e9473fac",
+"user":"benarent"
+}
+```
```

Comment:
This doesn't look like formatted JQ output, it's not indented properly etc.

Should be like this:
```
{
  "argv": [
    "google.com"
  ],
  "cgroup_id": 4294968064,
  "code": "T4000I",
  "ei": 5,
  "event": "session.command",
  "login": "root",
  "namespace": "default",
  "path": "/bin/ping",
  "pid": 2653,
  "ppid": 2660,
  "program": "ping",
  "return_code": 0,
  "server_id": "96f2bed2-ebd1-494a-945c-2fd57de41644",
  "sid": "44c6cea8-362f-11ea-83aa-125400432324",
  "time": "2020-01-13T18:05:53.919Z",
  "uid": "734930bb-00e6-4ee6-8798-37f1e9473fac",
  "user": "benarent"
}
```